doc/_static/zotero.bib

```diff
@@ -1033,6 +1033,18 @@ @inproceedings{gnu-parallel-2024
   doi = {10.1109/SCW63240.2024.00257}
 }
 
+@article{graham-efficientalgorithm-1972,
```

Comment:
Early you had the key as: `graham-convexhull-1972`

Reply:
Oops this is a zotero sync issue, will fix.